cmd/migration-managerd/internal/api/workers.go

```diff
@@ -806,7 +826,7 @@ func (d *Daemon) finalizeCompleteInstances() bool {
 			t, err := d.target.GetByID(ctx, *i.TargetID)
 			if err != nil {
 				log.Warn("Failed to get target", slog.Int("target_id", *i.TargetID), logger.Err(err))
-				_, err := d.instance.UpdateStatusByUUID(ctx, i.UUID, api.MIGRATIONSTATUS_ERROR, err.Error(), true)
+				_, err := d.instance.UpdateStatusByID(ctx, i.UUID, api.MIGRATIONSTATUS_ERROR, err.Error(), true)
```

Comment:
We don't want to undo the method renaming. :wink:

Reply:
Oops, bad rebase between commits :)